docs/site/Version-generator.md

```diff
@@ -0,0 +1,64 @@
+---
+lang: en
+title: 'Version generator'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Version-generator.html
+---
+
+{% include content/generator-create-app.html lang=page.lang %}
+
+### Synopsis
+
+The `lb4 version` command runs against a LoopBack 4 project and checks
```

Comment:
I find the name `version` problematic, it does not communicate the fact that it can be used to update dependencies. We already have `lb4 --version` to print the version of LoopBack dependencies installed in the project.

I am proposing:

(1)
Improve `lb4 --version` to include information about packages that are outdated. At the moment, we print output like this:

```
$ lb4 --version
@loopback/cli version: 1.24.0

@loopback/* dependencies:
  - @loopback/authentication: ^3.2.1
  - @loopback/boot: ^1.5.10
...
```

I am proposing to add a short text after each outdated version, possibly highlighted using `chalk.red`:

```
$ lb4 --version
@loopback/cli version: 1.24.0

@loopback/* dependencies:
  - @loopback/authentication: ^3.0.1 (outdated)
  - @loopback/boot: ^1.5.10
```

(2)
Rename the new command to something like `lb4 update` or `lb4 update-deps`. Personally, I prefer `lb4 update`, because it's short and in line with `npm update` command that most developers are hopefully familiar with.

This opens a question though - what is the benefit of this LB-specific update command when compared to `npm update`? Why (or when?) should our users use this new command instead of `npm update`?

Reply:
> What is the benefit of this LB-specific update command when compared to npm update? Why (or when?) should our users use this new command instead of npm update?

It would be awesome if `lb4 update` could run scripts to automatically apply changes necessary to migrate the codebase to newer conventions. That would be a perfect explanation why to run `lb4 update` instead of `npm update`.

Reply:
I'm good to have `lb4 update`. The key benefit of `lb4 update` over `npm update` is that `lb4 update` honors the collection of compatible LB4 packages that released with the current `lb4` command from `@loopback/cli`.

Reply:
Please `--version` is implemented at `cli.js` level without a yeoman generator.